### ISSUE TITLE
datasets of SmolVLM

### ISSUE THREAD

**Issue by xushilin1** [huggingface/smollm#73]
Thank you for your outstanding contribution to SmolVLM. 
You've mentioned in the paper that the data will be made public. Could you kindly inform me of its location? 
Additionally, I would appreciate it if you could provide more insight into the training process, including aspects like learning rate and various stages of training, among others.

**Comment by andimarafioti** [comment]
Hi! All the configs for training are available here: https://github.com/huggingface/smollm/tree/main/vision/experiments/pretraining/vloom
For example, for the 256M model the config is this one: https://github.com/huggingface/smollm/blob/main/vision/experiments/pretraining/vloom/tr_341_smolvlm_025b_1st_stage/config.yaml

For the datasets, in those folders you find slurm scripts that point to the datasets we used:
https://github.com/huggingface/smollm/blob/main/vision/experiments/pretraining/vloom/tr_341_smolvlm_025b_1st_stage/train.slurm
Everything is available publicly (Laion COCO, CM4, etc). The SFT mixes are just the cauldron with some data selection + a few datasets we added like Docmatix. 
We are working on a new SFT dataset, sort of a Cauldron V2 and that one should come with filtered versions from the get go! It's an important step that we didn't share about properly before
